src/plugins/LADTable/components/LADRow.vue

```diff
@@ -102,30 +107,31 @@ export default {
     methods: {
         updateValues(datum) {
             let newTimestamp = this.formats[this.timestampKey].parse(datum),
-                update = false, limit;
+                shouldUpdate = false,
+                limit;
 
             if(this.inBounds(newTimestamp)) {
 
                 // if timestamp is set, need tocheck, else update
                 if(this.timestamp !== '---') {
-                    let existingTimestamp = this.formats[this.timestampKey].parse(this.timestamp);
+                    let existingTimestamp = this.formattedTimestamp;
 
                     // if existing is in bounds, need to check, if not update
                     if(this.inBounds(existingTimestamp)) {
```

Comment:
We're passing a formatted value into `this.inBounds()`. That doesn't seem right, since that function compares the given value to the current bounds, which are numbers. I suspect that what we want to pass in here is `newTimestamp`?

Reply:
we do the new time stamp above, I think I need to just parse existing timestamp before passing it in... I'll look into this.

Reply:
Actually, looks like we don't need existingTimestamp at all, it was from before we switched to computed formattedTimestamp value. In this case we should just be able to use this.timestamp as it stores the parsed existing value.